doc/plutus/tutorials/basic-apps.rst

```diff
@@ -148,51 +141,195 @@ All that's missing now is the code for retrieving the funds, and some glue to pu
-The other two constraints use ``mustPayToPubKey`` to add payments for the recipients.
+:hsobj:`Ledger.Constraints.collectFromTheScript` takes the script outputs in ``unspentOutputs`` and adds them as input to the transaction, using the unit ``()`` as the redeemer.
+The other two constraints use :hsobj:`Ledger.Constraints.mustPayToAddress` to add payments for the recipients.
 
-Deploying the app on the Playground
------------------------------------
+Running the app on the Plutus apps emulator
+-------------------------------------------
 
 You have all the functions you need for the on-chain and off-chain parts of the app.
-Every contract in the Playground must define its public interface like this:
+Every contract in the Plutus apps emulator must define its public interface like this:
 
 .. literalinclude:: BasicApps.hs
    :start-after: BLOCK9
    :end-before: BLOCK10
 
-The Playground server uses the ``endpoints`` definition to populate the UI (via the schema, in our case ``SplitSchema``) and to start the simulation.
-``endpoints`` is the high-level definition of our app:
+``splitPlutusApp`` is the high-level definition of our app:
 
 .. literalinclude:: BasicApps.hs
    :start-after: BLOCK10
    :end-before: BLOCK11
 
-The ``select`` function acts like a choice between two branches.
+The :hsobj:`Plutus.Contract.selectList` function acts like a choice between two branches.
 The left branch starts with ``lock`` and the right branch starts with ``unlock``.
 The app exposes both endpoints and proceeds with the branch that receives an answer first.
 So, if you call the ``lock`` endpoint in one of the simulated wallets, it will call ``lockFunds`` and ignore the ``unlock`` side of the contract.
+The ``forever`` call, which runs the application in an infinite loop, is necessary for the
+:hsobj:`Plutus.Trace.Emulator.EmulatorTrace`.
+If you omit it, you will only be able to trigger a single endpoint after activating the contract,
+from which the contract instance will close.
 
-You also need a couple of declarations that generate glue code for the Playground.
-
-.. code-block:: haskell
-
-   mkSchemaDefinitions ''SplitSchema
-
-   $(mkKnownCurrencies [])
-
-You also need an additional import at the top of the file.
-
-.. code-block:: haskell
-
-   import Playground.Contract
-
-After that, you can compile the contract and create a simulation.
+You can now compile the contract and create a simulation.
 The following action sequence results in two transactions that lock the funds and then distribute them to the two recipients.
 
-.. figure:: images/playground-basic-app-simulation.png
+.. literalinclude:: BasicApps.hs
+   :start-after: BLOCK11
+   :end-before: BLOCK12
+
+You should see an output similar to what follows::
+
+  [INFO] Slot 0: TxnValidate 43ba666cc8a22a04b63a3b605ce14146dfa5ed999986625ad90c1bc16dabdd84 [  ]
```

Comment:
Make this a golden test output and include the file here? Not sure how much trouble that is.

Reply:
Good idea!

Done. Was fairly quick.